src/components/TaskItem/TaskItem.tsx

```diff
@@ -5,58 +5,63 @@ import { AiFillEdit } from 'react-icons/ai';
 import IconButton from '../IconButton/IconButton';
 import clsx from 'clsx';
 import styles from './TaskItem.module.scss';
-import UseClickOutside from '../../hooks/useClickOutside';
+import useClickOutside from '../../hooks/useClickOutside';
+import { TaskItemType } from '../../types/TaskItemType';
+import { useTaskStorage } from '../../hooks/useTaskStorage';
```

Comment:
useTaskStorege() should be ONE - Should NOT be used anywhere besides place where main taskList is extracted(ToDoSection). Remember boxes and cars my friend 🚗  These consume lots of fuel and even though theire on the same road, each ride on different lane and theire not in contact with eachother